### ISSUE TITLE
Kind-CLI image

### ISSUE THREAD

**Issue by pehli025** [kubernetes-sigs/kind#3383]
**What would you like to be added**:
Would it be possible to get a kind-cli image?

**Why is this needed**:
I am using kind as a way for testing in our git lab pipelines and it would be a more lightweight on the pipelines.

**Comment by BenTheElder** [comment]
See past issues discussing this, kind is available as a static go binary and should be run on the host if at all possible.

We're not planning to host an image, you can copy the binary into an empty image even an it will work as it has no external dependencies (c libraries etc) aside from docker/podman/the host kernel etc., but with all of the problems of attempting to containerize kind.

**Comment by kuwv** [comment]
> See past issues discussing this, kind is available as a static go binary and should be run on the host if at all possible.
> 
> We're not planning to host an image, you can copy the binary into an empty image even an it will work as it has no external dependencies (c libraries etc) aside from docker/podman/the host kernel etc., but with all of the problems of attempting to containerize kind.

@BenTheElder could you rephrase? I can't make out what you are saying specifically.

But, if I understand correctly, we're aware of various installation methods. We were using Alpine and the maintainer just broke that method.

IMO, kind as a powerful CI/CD tool. Having kind support DIND more natively just seems like a natural evolution.

 This would just be helpful. Anyway, thank you for your consideration. I'll try looking at the previous conversations.
